CoreFoundation/Parsing.subproj/CFXMLInterface.c

```diff
@@ -33,6 +33,13 @@ CF_INLINE struct _NSCFXMLBridgeStrong __CFSwiftXMLParserBridgeGetStronglyTyped()
 }
 #define __CFSwiftXMLParserBridgeCF (__CFSwiftXMLParserBridgeGetStronglyTyped())
 
+// CFSTR is not supported on Windows, so we have to fake it just for this file
```

Comment:
This comment is just wrong.  `CFSTR` is absolutely supported on Windows.  It is just not possible to create a reference to *PRIVATE* symbols of another module on Windows.  I would say that we want the same behaviour on all platforms - just because it happens to link does not make it proper behaviour.

Reply:
Ok so I can revert this file then.